examples/src/examples/graphics/shadow-cascades.tsx

```diff
@@ -79,7 +95,7 @@ class ShadowCascadesExample extends Example {
 
             // create the primitive using the material
             const primitive = new pc.Entity();
-            primitive.addComponent('render', {
+            primitive.addComponent('model', {
```

Comment:
How come you're switching from render to model?

Reply:
ah yeah .. I fixed the orbit camera to work with the render component in different PR that is not merged yet. Switching back to render. Thank God for your good eyes.